src/main/CommandHandler.cpp

```diff
@@ -167,6 +167,15 @@ CommandHandler::fileNotFound(std::string const& params, std::string& retStr)
 void
 CommandHandler::manualClose(std::string const& params, std::string& retStr)
 {
+    ZoneScoped;
+    std::map<std::string, std::string> retMap;
+    http::server::server::parseParams(params, retMap);
+
+    if (!retMap.empty())
+    {
+        mApp.manualClose(retMap, retStr);
```

Comment:
we should really share the code with manual close (I think this will become evident if we use virtual time).
I would just expect the behavior to be additive: for example, if we need to "jump ahead" the ledger seq num, we would do that before performing the existing "manualClose" behavior, same thing with the close time.

Reply:
If I understand what you're suggesting correctly, then this is done now.  `Application::manualClose()` is now a single interface with optional parameters for the sequence number and close time, and it's called in just one place from `CommandHandler::manualClose()`.